File: 1301-1400/1344-Angle Between Hands of a Clock/1344-Angle Between Hands of a Clock.cpp
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double ha = hour % 12 * 360 / 12 + (double)minutes / 60 * 30;
        double ma = minutes * 360 / 60;
        return min(abs(ha - ma), 360 - abs(ha - ma));
    }
};

File: 1301-1400/1344-Angle Between Hands of a Clock/1344-Angle Between Hands of a Clock.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        ha = hour % 12 * 360 / 12 + minutes / 60 * 30
        ma = minutes * 360 / 60
        return min(abs(ha - ma), 360 - abs(ha - ma))
